src/GraphQL/Types/Fields/FieldType.cs

```diff
@@ -75,5 +75,10 @@ internal void SetName(string name, bool validate)
         /// Gets or sets a field resolver for the field. Only applicable to fields of output graph types.
         /// </summary>
         public IFieldResolver? Resolver { get; set; }
+
+        /// <summary>
+        /// Gets or sets a subscription resolver for the field. Only applicable to subscription fields.
```

Comment:
It's worth to add additional check into schema validation.

Reply:
I agree

Reply:
- #2994